src/util/Util.js

```diff
@@ -16,7 +16,7 @@ let deprecationEmittedForRemoveMentions = false;
  */
 class Util extends null {
   /**
-   * Flatten an object. Any properties that are collections will get converted to an array of keys.
+   * Static method to flatten an object. Any properties that are collections will get converted to an array of keys.
```

Comment:
We don't use the words "static" and "method" to describe a... static method.

Instead, the following is more grammatically correct:
```suggestion
   * Flattens an object out. Any properties that are collections will get converted to an array of keys.
```